Zend/tests/match/block_expr_no_result.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Match expression block must return a value
+--FILE--
+<?php
+var_dump(match (1) {
+    1 => {
+        echo "Not returning anything\n";
+    },
+});
+?>
+--EXPECTF--
+Fatal error: Blocks of match expression with used result must return a value. Did you mean to omit the last semicolon? in %s on line %d
```

Comment:
I believe the “Did you mean […]” part is inconsistent with any other error message. It is useful, but consistency also is.

Reply:
We have a few of those, if you search for "Did you mean" in the code base.

Reply:
Oh, indeed there exactly two of those. I don't think I've ever seen the “int vs integer” one and it's been a long time since I've seen the “continue targets switch” one. So, point that it feels odd still stands :grin:

Reply:
There's also:

https://github.com/php/php-src/blob/d7eb4cfdb20d747cbe74aefcb6ebd4c31a092131/Zend/zend_compile.c#L2574-L2575

https://github.com/php/php-src/blob/d7eb4cfdb20d747cbe74aefcb6ebd4c31a092131/Zend/zend_compile.c#L2595-L2596

Just formatted differently.

Reply:
I change the syntax, so this is no longer relevant.